src/tfr_navigation/include/tfr_navigation/navigator.h

```diff
@@ -0,0 +1,58 @@
+/**
+ *  Controls the business logic of the navigation subsystem. 
+ *  Mainly it accepts commands to go to specific locations, and manages the 
+ *  goal of the navigation stack at a set interval to get it there.
+ *
+ *  Publishes and subscribes to the standard topics for a navigation action
+ *  server.
+ *
+ *  Relevant Messages: Navigation[Goal|Feedback|Result]
+ * */
+#ifndef NAVIGATOR_H
+#define NAVIGATOR_H
+#include<ros/ros.h>
+#include<ros/console.h>
+#include<actionlib/server/simple_action_server.h>
+#include<navigation_goal_manager.h>
+#include<tfr_msgs/NavigationAction.h>
+#include<nav_msgs/Odometry.h>
+#include<boost/bind.hpp>
+#include<cstdint>
```

Comment:
There are some missing spaces here. Doesn't cause any errors, but it does look weird.

Reply:
done!